### DISCUSSION TITLE
V2, V3 price quotes

### DISCUSSION THREAD

**Discussion by ars3nicum** [uniswap-python/uniswap-python#171] [Q&A] [answered]
For some tokens I get an error when quoting the price in Uniswap V3, meanwhile in V2 everything is OK.
Error code is:
raise ContractLogicError('execution reverted')
web3.exceptions.ContractLogicError: execution reverted

Please help me to understand the reason.

**Comment by liquid-8** [comment, marked as answer]
Check if there is proper pool @ V3. Check what fee has been set for it. Default value for V3 fee is 0.3% or fee=3000 for get_price/make_trade calls. Other values are 0.05% (fee=500) and 1% (fee=10000)

**Comment by ars3nicum** [comment]
It was fee. Thank you!

**Reply by yudidi** [reply]
it works both on ethereum and sepolia? I only success on ethereum,but meet 'execution reverted' too on sepolia.
